news/_posts/public-oss-demo/2022-07-06-public-demo-5-2022.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  "PrestaShop Project Public Demo - July 2022 session"
+subtitle: "We held the fifth session of this year open source project's public demo"
+date:   2022-07-06 22:00
```

Comment:
I noticed many recent deployment failures and I wonder if there is an offset between the time we write there (GMT) and the time used by GitHub

And if, when we publish, GitHub acknowledges this is not the past, it does not publish.

Reply:
Yes maybe 😅 

It's a bit "clicking on random button" but it's safer

Reply:
Ok, I've just changed the time, feel free to merge it if it is ok

Reply:
let's see if it works :p

Reply:
it works 🎉 thanks:)